content/open-source/octo-sts/overview.md

```diff
@@ -0,0 +1,119 @@
+2. **Define trust policies**: Create policy files that specify which identities can access which resources
+3. **Exchange tokens**: Workloads present OIDC tokens to Octo STS
+4. **Receive GitHub tokens**: If the identity matches the trust policy, Octo STS issues a short-lived GitHub token with specified permissions
+
+### The Token Exchange Process
+
+```
+[Workload] → OIDC Token → [Octo STS] → Validates → Trust Policy
+                                          ↓
+                            Short-lived GitHub Token ← Returns
+```
+
+The workload can then use this GitHub token to interact with the GitHub API, open pull requests, push commits, or perform other operations as defined by the trust policy.
+
+## Key Features
+
+**OIDC-Based Federation**
+Octo STS validates OIDC tokens from various identity providers, including GitHub Actions, cloud provider identity services, and Kubernetes service accounts.
+
+**Fine-Grained Access Control**
+Trust policies specify exactly which permissions to grant, following the principle of least privilege. You can create different policies for different workloads.
+
+**Pattern Matching**
+Trust policies support both exact matching and regular expressions for flexible identity validation, including custom claim matching.
+
+**Repository and Organization Scope**
+Policies can be scoped to individual repositories or shared across an organization, providing flexibility in how you manage access.
+
+**Automatic Token Expiration**
+All tokens issued by Octo STS are short-lived, reducing the window of opportunity if a token is compromised.
+
+## Common Use Cases
+
+**Automated Dependency Updates**
+Run tools like Renovate as GitHub Actions without storing PATs, automatically opening pull requests to update dependencies.
+
+**CI/CD Pipelines**
+Enable continuous integration and deployment workflows from cloud environments or Kubernetes clusters to interact with GitHub securely.
+
+**GitOps Workflows**
+Allow Kubernetes operators and controllers to update repository contents without long-lived credentials.
+
+**Cloud-to-GitHub Integration**
+Connect cloud workloads running in AWS, GCP, or Azure to GitHub repositories using native cloud identity.
+
+**Multi-Repository Automation**
+Create organization-level policies that allow automation tools to work across multiple repositories with appropriate permissions.
+
+## Security Model
+
+Octo STS implements several security best practices:
+
+- **Branch protection enforcement**: Policies cannot bypass repository branch protection rules
+- **Audit trail**: All token exchanges are logged for security monitoring
+- **Minimal permissions**: The app requests a superset of permissions but only grants what's specified in trust policies
+- **Regular permission reviews**: The Octo STS team updates permissions quarterly with community notification
+
+## Getting Started
```

Comment:
This feels kind of redundant with the "How Octo STS Works" section above

Reply:
Agreed, I removed it.